docs/extend/modular-resources/configure.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Configure your module and modular resource"
+linkTitle: "Configure"
+weight: 30
+type: "docs"
+tags: ["server", "rdk", "extending viam", "modular resources", "components", "services"]
+description: "Use the Viam module system to implement custom resources that can be included in any Viam-powered robot."
+no_list: true
+---
+
+Configure a module on your robot to make one or more modular resources available for configuration.
+
+If you want to create your own modular resource, you must first follow [these steps](/extend/modular-resources/create/) to code your own module and generate an executable.
+If you are using a pre-built modular resource, make sure you install the module and determine the filename of [the module's executable](/extend/modular-resources/create/#compile-the-module-into-an-executable).
+
+Follow these steps to configure the module and modular resource:
+
+1. [Save the executable](#make-sure-viam-server-can-access-your-executable) in a location your `viam-server` instance can access.
+2. [Add a **module**](#configure-your-module) referencing this executable to the configuration of your robot.
+3. [Add a new component or service](#configure-your-modular-resource) referencing the custom resource provided by the configured **module** to the configuration of your robot.
+
+{{% alert title="Modules vs. modular resources" color="tip" %}}
```

Comment:
This tip may belong on the "key concepts" page

Reply:
It's covered there as well, I just thought it helps in here as it's so confusing

Reply:
I generally continue to be confused, so +1 from me to retain!

Reply:
Okay fine to retain here but where is it covered on the key concepts page? I don't see it covered there.

Reply:
Oops I thought I put that alert in there too but I didn't keep it! adding